.markdownlinkcheck.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ignorePatterns": [{
+        "pattern": "^https://calendar.google.com/calendar"
```

Comment:
not friendly to being crawled with this tool, maybe some useragent or header issue